tasks/install_common.yml

```diff
@@ -1,13 +1,15 @@
 ---
 - name: Create the /tmp/efs-utils directory
   file:
+    mode: 0644
```

Comment:
This mode should be `0755`, right?

Reply:
Good catch - fixed in https://github.com/cisagov/ansible-role-amazon-efs-utils/pull/7/commits/eb174f06a35ad96e8a0489a17cfd9545ac65e3d3.

Reply:
Actually, you changed it in the wrong place.  I will fix it.

Reply:
Arg- sorry about that!! 🤬

Reply:
See commit da6dd24.